modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1386,6 +1441,35 @@ public void testDeleteObjectDefinition() throws Exception {
 				ObjectDefinition.class.getName(),
 				ResourceConstants.SCOPE_INDIVIDUAL,
 				String.valueOf(objectDefinition.getObjectDefinitionId())));
+
+		// Modifiable system object definition deletion
```

Comment:
Not sure, but I believe that this message is not necessary.

Reply:
We have a similar case [here](https://github.com/liferay/liferay-portal/blob/eb89549401eb2d0bdab3683aecd59649a867fd2a/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java#L1177), I believe would be better to leave the comment here